torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -142,7 +142,7 @@ template std::shared_ptr<OwnerRRef<py::object>> RRefContext::
 
 template <typename T>
 std::shared_ptr<OwnerRRef<T>> RRefContext::createOwnerRRef() {
-  // Don't add this OnwerRRef to the owners_ map yet, otherwise
+  // Don't add this OnwerRRef to the owners_ map yet by default, otherwise
```

Comment:
is this change in the comment still pertinent in the latest version?

Reply:
This change should be removed, let me update. Thanks!